content/contribute/infrastructure.md

```diff
@@ -112,6 +112,7 @@ Expand the tab below to see an annotated `tree` output of the website repository
 
 {{<expand >}}
 ```shell
+tree .
```

Comment:
Need the copy only first line here too?

Reply:
I don't think so since it's just illustrative of what the contents are and there aren't instructions provided around what directory you're in.

Reply:
Makes sense! Thanks for the context